inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -23,34 +23,36 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
 };
 
 const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes = {
-        {Sigmoid,     {}},
-        {Tanh,        {}},
-        {Relu,        {}},
-        {Exp,         {}},
-        {Log,         {}},
-        {Sign,        {}},
-        {Abs,         {}},
-        {Clamp,       {{-2.0f, 2.0f}}},
-        {Negative,    {}},
-        {Acos,        {}},
-        {Asin,        {}},
-        {Atan,        {}},
-        {Cos,         {}},
-        {Cosh,        {}},
-        {Floor,       {}},
-        {Sin,         {}},
-        {Sinh,        {}},
-        {Sqrt,        {}},
-        {Tan,         {}},
-        {Elu,         {{0.1f}}},
-        {Erf,         {}},
-        {HardSigmoid, {{0.2f, 0.5f}}},
-        {Selu,        {{1.6732f, 1.0507f}}},
-        {Ceiling,     {}},
-        {Mish,        {}},
-        {HSwish,      {}},
-        {SoftPlus,    {}},
-        {HSigmoid,    {}}
+        {Sigmoid,               {}},
+        {Tanh,                  {}},
+        {Relu,                  {}},
+        {Exp,                   {}},
+        {Log,                   {}},
+        {Sign,                  {}},
+        {Abs,                   {}},
+        {Clamp,                 {{-2.0f, 2.0f}}},
+        {Negative,              {}},
+        {Acos,                  {}},
+        {Asin,                  {}},
+        {Atan,                  {}},
+        {Cos,                   {}},
+        {Cosh,                  {}},
+        {Floor,                 {}},
+        {Sin,                   {}},
+        {Sinh,                  {}},
+        {Sqrt,                  {}},
+        {Tan,                   {}},
+        {Elu,                   {{0.1f}}},
+        {Erf,                   {}},
+        {HardSigmoid,           {{0.2f, 0.5f}}},
+        {Selu,                  {{1.6732f, 1.0507f}}},
+        {Ceiling,               {}},
+        {Mish,                  {}},
+        {HSwish,                {}},
+        {SoftPlus,              {}},
+        {HSigmoid,              {}},
+        {RoundHalfToEven,       {}},
+        {RoundHalfAwayFromZero, {}}
```

Comment:
One more thing about tests. The main difference for rounding algorithms is how they handle *.5 values (0.5, 1.5, -0.5 etc). Currently ActivationLayerTest randomizes input data in floating point range and it is highly unlikely that it will produce exactly *.5 values. I think  we should update GenerateInput for Round algorithms in order to provide such numbers. @iefode What do you think?
We can merge the PR as is, but need to consider this in future.

Reply:
I saw [similar changes](https://github.com/openvinotoolkit/openvino/pull/2575/files#diff-61747879b02e692cb4482fd43de619370da43590132606577ca21466c23ef790) from GNA team. I guess we can move it into `activation.cpp` and reuse this functionality for your purposes. @dmitry-gorokhov @bsochack Do you agree to use such approach?

Reply:
I agree.
@a-sidorova Please create follow up ticket to cover .5 cases for Round operation and link in with Issue 29090